corehq/apps/app_manager/views/modules.py

```diff
@@ -827,6 +827,11 @@ def upgrade_shadow_module(request, domain, app_id, module_unique_id):
 def overwrite_module_case_list(request, domain, app_id, module_unique_id):
     app = get_app(domain, app_id)
     dest_module_unique_ids = request.POST.getlist('dest_module_unique_ids')
+    src_module = app.get_module_by_unique_id(module_unique_id)
+    detail_type = request.POST['detail_type']
+
+    # For short details, user selects which properties to copy.
```

Comment:
Someday someone will read this and be grateful. Thanks for adding documentation.